gio/src/output_stream.rs

```diff
@@ -65,6 +65,79 @@ pub trait OutputStreamExtManual: Sized + OutputStreamExt {
         >,
     >;
 
+    #[cfg(any(feature = "v2_60", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_60")))]
+    #[doc(alias = "g_output_stream_writev")]
+    fn writev(
+        &self,
+        vectors: &[&[u8]],
+        cancellable: Option<&impl IsA<Cancellable>>,
+    ) -> Result<usize, glib::Error>;
+
+    #[cfg(any(feature = "v2_60", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_60")))]
+    #[doc(alias = "g_output_stream_writev_async")]
+    fn writev_async<
+        B: AsRef<[u8]> + Send + 'static,
+        P: FnOnce(Result<(Vec<B>, usize), (Vec<B>, glib::Error)>) + 'static,
```

Comment:
Maybe can be made generic instead of using `Vec` here

Reply:
Same applies to other functions here

Reply:
How? It needs to be an owned type so it can be held by the closure/future.